src/theia/sfm/bundle_adjustment/bundle_adjustment.cc

```diff
@@ -184,14 +178,20 @@ BundleAdjustmentSummary BundleAdjustPartialReconstruction(
     }
 
     Camera* camera = view->MutableCamera();
-    // This function will add all camera parameters to the problem and will keep
-    // the intrinsic params constant if desired.
-    AddCameraParametersToProblem(constant_intrinsics,
-                                 camera->mutable_parameters(),
+
+    problem.AddParameterBlock(camera->mutable_extrinsics(), Camera::kExtrinsicsSize);
+
+    // This function will add intrinsic camera parameters to the problem and will keep
```

Comment:
Theia uses the Google c++ style guide which limits each line to 80-characters maximum. Please wrap this comment to conform.